src/styles/stopwatch.scss

```diff
@@ -0,0 +1,73 @@
+@import "./variables.scss";
+
+.stopwatch {
+  position: relative;
+  height: 100vh;
+
+  &__arrow-minutes {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
```

Comment:
You've used this logic for other arrow (as well as transform), let's create a placeholder for it